src/views/styles/user.scss

```diff
@@ -0,0 +1,3 @@
+.wrap-user{
+    background-color: red;
+}
```

Comment:
Nếu CSS này chỉ dành cho mỗi `user` module thì mình nên đặt tên nó theo syntax `[name].module.scss` để enable tính năng Component-Level CSS

Ref: https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css